File: Day21/DecimalToBinary.cpp
#include<iostream>
using namespace std;

void decimalToBinary(int decimal){
    int binary[32];
    int i = 0;
    
    while(decimal > 0){
        binary[i] = decimal % 2;
        decimal /= 2;
        i++;
    }
    for(int j= i-1; j>= 0; j--){
        cout<<binary[j];
    }
}
int main(){
    int decimal;
    cout<<"Enter the Decimal number: ";
    cin>>decimal;
    cout<<"Binary representation : ";
    decimalToBinary(decimal);
    return 0;
}

File: Day21/DecimalToBinary.java
package Day21;
import java.util.Scanner;
public class DecimalToBinary {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter the Decimal number: ");
        int decimal = sc.nextInt();
        sc.close();
        String binary = " ";

        while(decimal > 0){
            binary = (decimal % 2) + binary;
            decimal /= 2;
        }

        System.out.println("Binary representation : " + binary);

    }
}
